Fails to compile on Apple Silicon when running `cargo install just`

Am trying to use as a dep elsewhere, but am unable to install :( 

```
error[E0308]: mismatched types
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ctrlc-3.4.3/src/platform/unix/mod.rs:50:33
    |
50  | ...   .and_then(|_| fcntl(pipe.0, FcntlArg::F_SETFD(FdFlag::FD_CLOEXEC)))
    |                     ----- ^^^^^^ expected `i32`, found `OwnedFd`
    |                     |
    |                     arguments to this function are incorrect
    |
note: function defined here
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/nix-0.28.0/src/fcntl.rs:600:8
    |
600 | pub fn fcntl(fd: RawFd, arg: FcntlArg) -> Result<c_int> {
    |        ^^^^^

error[E0308]: mismatched types
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ctrlc-3.4.3/src/platform/unix/mod.rs:51:33
    |
51  | ...   .and_then(|_| fcntl(pipe.1, FcntlArg::F_SETFD(FdFlag::FD_CLOEXEC)));
    |                     ----- ^^^^^^ expected `i32`, found `OwnedFd`
    |                     |
    |                     arguments to this function are incorrect
    |
note: function defined here
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/nix-0.28.0/src/fcntl.rs:600:8
    |
600 | pub fn fcntl(fd: RawFd, arg: FcntlArg) -> Result<c_int> {
    |        ^^^^^

error[E0308]: mismatched types
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ctrlc-3.4.3/src/platform/unix/mod.rs:56:33
    |
56  | ...   .and_then(|_| fcntl(pipe.0, FcntlArg::F_SETFL(OFlag::O_NONBLOCK)))
    |                     ----- ^^^^^^ expected `i32`, found `OwnedFd`
    |                     |
    |                     arguments to this function are incorrect
    |
note: function defined here
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/nix-0.28.0/src/fcntl.rs:600:8
    |
600 | pub fn fcntl(fd: RawFd, arg: FcntlArg) -> Result<c_int> {
    |        ^^^^^

error[E0308]: mismatched types
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ctrlc-3.4.3/src/platform/unix/mod.rs:57:33
    |
57  | ...   .and_then(|_| fcntl(pipe.1, FcntlArg::F_SETFL(OFlag::O_NONBLOCK)));
    |                     ----- ^^^^^^ expected `i32`, found `OwnedFd`
    |                     |
    |                     arguments to this function are incorrect
    |
note: function defined here
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/nix-0.28.0/src/fcntl.rs:600:8
    |
600 | pub fn fcntl(fd: RawFd, arg: FcntlArg) -> Result<c_int> {
    |        ^^^^^

error[E0308]: mismatched types
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ctrlc-3.4.3/src/platform/unix/mod.rs:61:21
    |
61  |         Ok(_) => Ok(pipe),
    |                  -- ^^^^ expected `(i32, i32)`, found `(OwnedFd, OwnedFd)`
    |                  |
    |                  arguments to this enum variant are incorrect
    |
    = note: expected tuple `(i32, i32)`
               found tuple `(OwnedFd, OwnedFd)`
help: the type constructed contains `(OwnedFd, OwnedFd)` due to the type of the argument passed
   --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ctrlc-3.4.3/src/platform/unix/mod.rs:61:18
    |
61  |         Ok(_) => Ok(pipe),
    |                  ^^^----^
    |                     |
    |                     this argument influences the type of `Ok`
note: tuple variant defined here
   --> /Users/jamiepond/.rustup/toolchains/nightly-2023-10-26-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/result.rs:506:5
    |
506 |     Ok(#[stable(feature = "rust1", since = "1.0.0")] T),
    |     ^^

error[E0308]: mismatched types
    --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ctrlc-3.4.3/src/platform/unix/mod.rs:63:35
     |
63   |             let _ = unistd::close(pipe.0);
     |                     ------------- ^^^^^^ expected `i32`, found `OwnedFd`
     |                     |
     |                     arguments to this function are incorrect
     |
note: function defined here
    --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/nix-0.28.0/src/unistd.rs:1087:8
     |
1087 | pub fn close(fd: RawFd) -> Result<()> {
     |        ^^^^^

error[E0308]: mismatched types
    --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ctrlc-3.4.3/src/platform/unix/mod.rs:64:35
     |
64   |             let _ = unistd::close(pipe.1);
     |                     ------------- ^^^^^^ expected `i32`, found `OwnedFd`
     |                     |
     |                     arguments to this function are incorrect
     |
note: function defined here
    --> /Users/jamiepond/.cargo/registry/src/index.crates.io-6f17d22bba15001f/nix-0.28.0/src/unistd.rs:1087:8
     |
1087 | pub fn close(fd: RawFd) -> Result<()> {
     |        ^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `ctrlc` (lib) due to 7 previous errors
warning: build failed, waiting for other jobs to finish...

```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,7 +23,7 @@ atty = "0.2.0"
 blake3 = { version = "1.5.0", features = ["rayon", "mmap"] }
 camino = "1.0.4"
 clap = { version = "2.33.0", features = ["wrap_help"] }
-ctrlc = { version = "3.1.1", features = ["termination"] }
+ctrlc = { version = "=3.1.1", features = ["termination"] }
 derivative = "2.0.0"
 dirs = "5.0.1"
 dotenvy = "0.15"

